tests/python/operators/trace_test.py

```diff
@@ -0,0 +1,23 @@
+import numpy as np
+import time
+import ffsim
+
+def test_trace(norb = 50, nelec = (3,3)):
+    n = norb
+    
+    h = np.matrix(np.random.rand(n,n))
+    h = h + h.H
+    V = np.array([np.matrix(np.random.rand(n,n)),np.matrix(np.random.rand(n,n))])
+    H = ffsim.DiagonalCoulombHamiltonian(h,V)
```

Comment:
Just do
```python
H = ffsim.random.random_diagonal_coulomb_hamiltonian(norb, real=True, seed=rng)
```
where `rng` is your random Generator instance. I just added the `real` flag in https://github.com/qiskit-community/ffsim/pull/350 so you'll need to merge main first.